index.html

```diff
@@ -5,8 +5,23 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href = "styles/index.css" rel = "stylesheet"/>
 </head>
 <body>
-    
+    <main id = "main-section">
```

Comment:
no errors in the validator! great! there are a few warnings, but they are very specific and harmless